README.md

```diff
@@ -65,11 +66,30 @@ make up
 
 For more information, check out the documentation in the [presto_cluster](./example/presto_cluster) README.
 
+### Connect to the services (proxies)
+Since the services in this module uses the [`sidecar_service`](https://www.nomadproject.io/docs/job-specification/sidecar_service), you need to connect to the sevices using a [consul connect proxy](https://www.consul.io/commands/connect/proxy).
+The proxy connections are pre-made and defined in the `Makefile`:
+```sh
+make proxy-hive     # to hivemetastore
+make proxy-minio    # to minio
+make proxy-postgres # to postgres
+make proxy-presto   # to presto
+```
+
+You can now connect to Presto using the Presto CLI with the following command:
+```sh
+make presto-cli # connect to Presto CLI
+```
+
+#### :warning: Note
+If you are on a Mac the proxies and Presto CLI may not work.
+Instead, you can install the [Consul binary](https://www.consul.io/docs/install) and run the commands in the `Makefile` manually (without `docker run ..`).
+Further, you need to install the [Presto CLI](https://prestodb.io/) on your local machine or inside the box.
```

Comment:
Pay attention that `prestodb` is facebook distribution. We use opensource distribution `prestosql`.

Link to discussion https://github.com/prestosql/presto/issues/380 
```suggestion
Further, you need to install the [Presto CLI](https://prestosql.io/docs/current/installation/cli.html) on your local machine or inside the box.
```